models/src/main/scala/coop/rchain/models/SortedParMap.scala

```diff
@@ -11,14 +11,19 @@ case class SortedParMap(private val ps: Map[Par, Par])
 
   override def empty: SortedParMap = SortedParMap.empty
 
-  override def get(key: Par): Option[Par] = ps.get(key)
+  override def get(key: Par): Option[Par] = sortedMap.toMap.get(key)
```

Comment:
Let's just make `sortedMap: SortedMap[Par, Par]` instead of incurring the `toMap` overhead on every `get`.

Reply:
Ideally, the commit message would also mention that there was a possibility of accessing an un-sorted par via `get`/`apply` due to get relying on the unsorted `ps`. Which is actually the reason why the default equals implementation would say 'false'. Won't push on the commit message change though.

Reply:
Please though add a test securing the behavior of the `get` method as well. So the test should say 'when I lookup a value of the SortedMap using get, the returned value should be sorted'